frontend/src/components/UserSelectBox.jsx

```diff
@@ -0,0 +1,28 @@
+import { 
+    FormControl,
+    InputLabel,
+    Select,
+    MenuItem,
+} from '@mui/material';
+
+
+export  const UserSelectBox = ({users,selectedUser, handleChange}) =>{
+  return(
+  <>
+  <FormControl fullWidth>
+    <InputLabel id="user">User</InputLabel>
+    <Select
+      labelId="user"
+      id="user"
+      value={selectedUser}
+      label="User"
+      onChange={handleChange}
+    >
+    {users.map((user) => (
+      <MenuItem value = {user.id}>{user.username}</MenuItem>
```

Comment:
reactはmapなどを使って繰り返し要素を生成する場合に`key`というpropsを設定することが推奨されています。
```suggestion
      <MenuItem value={user.id} key={user.id}>{user.username}</MenuItem>
```
これは並び替えなどが発生した時などに使われる要素特定用のものでしてないといろいろ動かなくなってしまうことがあるためつけた方がいいです。
一意な値であるべきなのでidを使うと良いと思います